modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -717,6 +743,15 @@ protected DataDefinition testPatchDataDefinition_addDataDefinition()
 			testGroup.getGroupId(), _CONTENT_TYPE, dataDefinition);
 	}
 
+	@Override
+	protected DataDefinition testPostDataDefinitionCopy_addDataDefinition(
+			DataDefinition dataDefinition)
+		throws Exception {
+
+		return dataDefinitionResource.postSiteDataDefinitionByContentType(
+			testGroup.getGroupId(), _CONTENT_TYPE, dataDefinition);
```

Comment:
@devpedrotavares you should call the `postDataDefinitionCopy` instead.. the `testPostDataDefinitionCopy_addDataDefinition` should return the copied one..

Reply:
Hi! I did as you instructed. But could you please double-check that instruction? I noticed that this is not how it is currently being done in `com/liferay/object/admin/rest/resource/v1_0/test/ObjectViewResourceTest.testPostObjectViewCopy` and in `com/liferay/notification/rest/resource/v1_0/test/NotificationTemplateResourceTest.testPostNotificationTemplateCopy_addNotificationTemplate` for examples.